src/cdk/stepper/stepper.ts

```diff
@@ -67,7 +67,7 @@ export class StepperSelectionEvent {
   moduleId: module.id,
   selector: 'cdk-step',
   exportAs: 'cdkStep',
-  templateUrl: 'step.html',
+  template: '<ng-template><ng-content></ng-content></ng-template>',
```

Comment:
this wasn't necessary, right? the resource inlining has landed

Reply:
Wasn't necessary, just cleaned it up incidentally since the template was trivial.